index.html

```diff
@@ -289,6 +297,100 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+      <section class="contact-form">
+        <div class="sidebyside">
+          <h3 class="contact-text">
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h3>
+
+          <form
+            class="contact-elements"
+            action="https://formspree.io/f/mlezlyew"
+            method="POST"
+          >
+            <input
+              class="contact-name celement"
+              name="name"
+              type="text"
+              required
+              maxlength="30"
+              placeholder="Full name"
+            />
+
+            <input
+              class="contact-email celement"
+              name="email"
+              type="email"
+              required
+              placeholder="Email adress"
+            />
+
+            <textarea
+              class="contact-message celement"
+              name="message"
+              maxlength="500"
+              placeholder="Enter text here"
+              required
+            ></textarea>
+
+            <button class="btstyle mainbt send-message" type="button">
+              Get in touch
+            </button>
+          </form>
+        </div>
+
+        <footer class="foot-area">
+          <ul class="bottom-bar">
+            <li>
+              <a href="http://www.github.com">
+                <img
+                  class="contactbar-img footicon"
+                  src="./media/svg/github.svg"
+                  alt="github logo"
+                />
+              </a>
+            </li>
+            <li>
+              <a href="http://www.linkedin.com">
+                <img
+                  class="contactbar-img footicon"
+                  src="./media/svg/linkedin.svg"
+                  alt="linkedin logo"
+                />
+              </a>
+            </li>
+            <li>
+              <a href="https://angel.co">
+                <img
+                  class="contactbar-img footicon"
+                  src="./media/svg/angel.svg"
+                  alt="peace logo"
+                />
```

Comment:
Please try and have your project be as close to the figma design as possible. Your icons in the footer have a different color to the one in the design. Remember this demonstrates that you can successfully use a figma design in the making of your project.
Figma's design
<img width="187" alt="Screenshot 2022-09-16 at 01 28 22" src="https://user-images.githubusercontent.com/48546992/190525631-fef3a8e3-8033-4766-8fd9-1ba42b4233fe.png">

Your design:
<img width="757" alt="Screenshot 2022-09-16 at 01 36 11" src="https://user-images.githubusercontent.com/48546992/190525666-0e2e72e3-dd2f-4895-aece-f89b9f8466e0.png">